### PR TITLE
Sync-ing latest changes to kitchen - 20170704

### PR DESCRIPTION
**Pull request by shamsimam**

### CODE REVIEW

File: kitchen/project.clj [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
+;;       All Rights Reserved
+;;
+;;       THIS IS UNPUBLISHED PROPRIETARY SOURCE CODE OF
+;;       Two Sigma Investments, LP.
+;;
+;;       The copyright notice above does not evidence any
+;;       actual or intended publication of such source code.
+;;
 (defproject kitchen "0.1.0-SNAPSHOT"
   :dependencies [^{:voom {:repo "https://github.com/twosigma/jet.git" :branch "waiter-patch"}}
```
Comment (shamsimam) [inline kitchen/project.clj]: TODO: Need to update the jet version after the jet PR (https://github.com/twosigma/jet/pull/1) has been merged.

**Comment by shamsimam** [comment]
@dposada do you know what might trigger this message: https://travis-ci.org/twosigma/waiter/jobs/250384306#L628

**Comment by shamsimam** [comment]
Passing build: https://travis-ci.org/twosigma/waiter/builds/250460682

File: kitchen/bin/build-uberjar.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+# Usage: build-uberjar.sh
+#
+# Builds the kitchen uberjar.
+
+set -ev
+
+KITCHEN_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )"
+export PATH=${KITCHEN_DIR}/..:$PATH
```
Comment (dposada) [inline kitchen/bin/build-uberjar.sh]: Can we do this line in `run-integration-tests*.sh` instead of here, since this is inappropriate for local runs?

Comment (shamsimam) [inline kitchen/bin/build-uberjar.sh]: Done, waiting for build.

**Comment by shamsimam** [comment]
Passing build: https://travis-ci.org/twosigma/waiter/builds/250533854

**Pull request merged**